common/src/main/java/org/jboss/hal/testsuite/page/config/JMSBridgePage.java

```diff
@@ -0,0 +1,76 @@
+package org.jboss.hal.testsuite.page.config;
+
+import org.jboss.hal.testsuite.finder.Application;
+import org.jboss.hal.testsuite.finder.FinderNames;
+import org.jboss.hal.testsuite.finder.FinderNavigation;
+import org.jboss.hal.testsuite.fragment.shared.modal.ConfirmationWindow;
+import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+import org.jboss.hal.testsuite.page.Navigatable;
+import org.jboss.hal.testsuite.util.ConfigUtils;
+import org.jboss.hal.testsuite.util.Console;
+
+/**
+ * Abstraction over JMS Bridge UI configuration
+ *
+ * @author Jan Kasik <jkasik@redhat.com>
+ *         Created on 9/19/16.
+ */
+public class JMSBridgePage extends MessagingPage implements Navigatable {
```

Comment:
No need for `implements Navigatable` as it already extends `MessagingPage` implementing it, but ok.